skia-safe/src/utils/shadow_utils.rs

```diff
@@ -8,7 +8,12 @@ bitflags! {
         const TRANSPARENT_OCCLUDER = sb::SkShadowFlags_kTransparentOccluder_ShadowFlag as u32;
         #[allow(clippy::unnecessary_cast)]
         const GEOMETRIC_ONLY = sb::SkShadowFlags_kGeometricOnly_ShadowFlag as u32;
-        const ALL = Self::TRANSPARENT_OCCLUDER.bits() | Self::GEOMETRIC_ONLY.bits();
+        #[allow(clippy::unnecessary_cast)]
+        const DIRECTIONAL_LIGHT = sb::SkShadowFlags_kDirectionalLight_ShadowFlag as u32;
+        #[allow(clippy::unnecessary_cast)]
+        const CONCAVE_BLUR_ONLY = sb::SkShadowFlags_kConcaveBlurOnly_ShadowFlag as u32;
+        const ALL = Self::TRANSPARENT_OCCLUDER.bits() | Self::GEOMETRIC_ONLY.bits()
+            | Self::GEOMETRIC_ONLY.bits() | Self::CONCAVE_BLUR_ONLY.bits();
```

Comment:
It seems that`Self::DIRECTIONAL_LIGHT` is missing from `ALL` and `GEOMETRIC_ONLY` got duplicated..